ui/app/models/ldap/library.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { withFormFields } from 'vault/decorators/model-form-fields';
+import { withModelValidations } from 'vault/decorators/model-validations';
+
+const validations = {
+  name: [{ type: 'presence', message: 'Library Name is required.' }],
+};
+const formFields = [
+  'name',
+  'service_account_names',
+  'default_ttl',
+  'max_ttl',
+  'disable_check_in_enforcement',
+];
+
+@withModelValidations(validations)
+@withFormFields(formFields)
+export default class LdapLibraryModel extends Model {
+  @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
+
+  @attr('string', { label: 'Library Name' }) name;
+
+  @attr('string', {
+    editType: 'stringArray',
+    label: 'Accounts',
+    subText:
+      'The names of all the accounts that can be checked out from this set. These accounts must only be used by Vault, and may only be in one set.',
+  })
+  service_account_names;
+
+  @attr('number', {
+    editType: 'ttl',
+    label: 'Default Lease TTL',
+    detailsLabel: 'TTL',
+    helperTextDisabled: 'Vault will use the default lease duration',
```

Comment:
Missing period. Anything that is a sentence that is displayed in the UI should end in a period. Unless it's a title or field label indicating a name. I clarified this with design.

Reply:
Right thanks. I wish there was a way to automate this and hopefully new designs in Figma will have the correct punctuation because I've gotten in the habit of copy and pasting all the text to avoid typo's and other mistakes with typing it out.